tracer/src/Datadog.Trace/AppSec/SecuritySettings.cs

```diff
@@ -116,6 +116,11 @@ bool GetEnabledDefaultValue()
                                   .WithKeys(ConfigurationKeys.AppSec.MaxStackTraceDepth)
                                   .AsInt32(val => val >= 0)
                                   .GetValueOrDefault(32);
+
+            WafDebugEnabled = config
+                        .WithKeys(ConfigurationKeys.AppSec.WafDebugEnabled)
+                        .AsBool()
+                        .GetValueOrDefault(false);
```

Comment:
`GetValueOrDefault` shouldn't be used here, the correct approach is to pass the default value in `AsBool` 🤔 Looks like all the security settings are incorrect here, so they're reporting the incorrect values to telemetry...